apps/performance-tests/src/util/IModelUtilities.ts

```diff
@@ -37,4 +39,9 @@ class BackendTestIModelBuilder implements TestIModelBuilder {
     const spec = this._iModel.codeSpecs.getByName(codeSpecName).id;
     return new Code({ scope: scopeModelId, spec, value: codeValue });
   }
+
+  public async importSchema(schemaContentXml: string, schemaName: string): Promise<void> {
```

Comment:
maybe schema name should go before its content?

Reply:
I wanted to have this method compatible with `TestIModelBuilder` in `presentation-testing` because `BackendTestIModelBuilder` base class `TestIModelBuilder` has to be compatible with that one of the `presentation-testing`. But since this method is already incompatible with in `presentation-testing`'s `TestIModelBuilder`, I can swap the arguments. Done.